docs/en/Installing.md

```diff
@@ -30,8 +16,6 @@ For more configuration options see [Configuration](Configuration.md).
 
 ## Upgrading
 
-### Migrating from version 2.* SilverStripe installations
-
-This module replaces the built-in commenting system available in versions up to SilverStripe 2.4. To migrate from that you'll need to run `dev/build` after installing the module.
+### Migrating from version SilverStripe 2 installations
 
-You can do this via sake (`sake dev/build flush=1`) or via a web browser by visiting `http://yoursite.com/dev/build?flush=1`
+This module replaces the built-in commenting system available in SilverStripe 2. Running the `dev/build` task will automatically perform a migration if any comments from these old versions exist.
```

Comment:
We can probably remove 2.x migration support now. We should be concerned only with 3.x migrations

Reply:
I thought so too :) But there are a few select sites around that do make that jump in the (wider) community, I think. I thought it doesn't much hurt.
Can easily remove it, it's a pretty simple section of a `requireDefaultRecords` function body. 

We could of course remove the code and leave a docs note about making a manual SQL query (it's a simple table copy `PageComment` -> `Comment`).

Thoughts?

Reply:
They can then migrate from 2.x to 3.x with the existing docs and code in previous branches. We don't want to support migrating across more than one major version :)

Reply:
That is true.
I'll remove it now :)